### PR TITLE
fix(list-view): show pointer on hover of metadata column header

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- Take in isGridHeaderSortable handler that determines whether or not a column header is sortable
- Apply ```cursor: pointer``` if they column header is sortable

ezgif com-video-to-gif (6)

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ChenCodes has signed the CLA. Thanks for the pull request!

**Comment by macrotim** [comment]
Let's obsolete this with https://github.com/box/box-ui-elements/pull/1144

File: src/features/list-view/ListView.js [hunk 1]
```diff
@@ -22,33 +22,44 @@ type Props = {
...
             const displayName = getGridHeader(columnIndex);
             const sortDirection = getGridHeaderSort(columnIndex);
             const isSortAsc = sortDirection === SORT_ORDER_ASCENDING;
-            const className = classNames({
+            const iconClassName = classNames({
                 'bdl-ListView-isSortAsc': isSortAsc,
             });
+            const isHeaderSortable = isGridHeaderSortable(columnIndex);
 
+            const headerClassName = classNames('bdl-ListView-columnHeader', {
```
Comment (Visylvius) [inline src/features/list-view/ListView.js]: headerClassName seems generic, not descriptive enough.
Perhaps we could change this to something else?

Comment (ChenCodes) [reply src/features/list-view/ListView.js]: There's only one header within this ListView component so I think it's appropriate that we have a generic header class name. If there were multiple headers in this component then I would push for a more specific name.

File: src/features/list-view/ListView.js [hunk 2]
```diff
@@ -22,33 +22,44 @@ type Props = {
...
+                'bdl-ListView-isHeaderSortable': isHeaderSortable,
+            });
             return (
                 <button
-                    className="bdl-ListView-columnHeader"
+                    className={headerClassName}
```
Comment (macrotim) [inline src/features/list-view/ListView.js]: Should be "buttonClassName".

File: src/features/list-view/ListView.js [hunk 3]
```diff
@@ -22,33 +22,44 @@ type Props = {
...
                     key={key}
                     style={style}
                     type="button"
                     onClick={() => onSortChange(columnIndex)}
```
Comment (macrotim) [inline src/features/list-view/ListView.js]: Please add... isHeaderSortable  && onSortChange(columnIndex)

**Pull request merged**